src/codeowners.md

```diff
@@ -0,0 +1,199 @@
+  * Swing: [`swing-dev@openjdk.java.net`](mailto:swing-dev@openjdk.java.net)
+* Core Libs: [`core-libs-dev@openjdk.java.net`](mailto:core-libs-dev@openjdk.java.net)
+  * Net: [`net-dev@openjdk.java.net`](mailto:net-dev@openjdk.java.net)
+  * NIO: [`nio-dev@openjdk.java.net`](mailto:nio-dev@openjdk.java.net)
+* HotSpot: [`hotspot-dev@openjdk.java.net`](mailto:hotspot-dev@openjdk.java.net)
+  * Compiler: [`hotspot-compiler-dev@openjdk.java.net`](mailto:hotspot-compiler-dev@openjdk.java.net)
+  * GC: [`hotspot-gc-dev@openjdk.java.net`](mailto:hotspot-gc-dev@openjdk.java.net)
+  * Runtime: [`hotspot-runtime-dev@openjdk.java.net`](mailto:hotspot-runtime-dev@openjdk.java.net)
+  * Serviceability: [`serviceability-dev@openjdk.java.net`](mailto:serviceability-dev@openjdk.java.net)
+* I18n: [`i18n-dev@openjdk.java.net`](mailto:i18n-dev@openjdk.java.net)
+* Langtools
+  * Amber Project: [`amber-dev@openjdk.java.net`](mailto:amber-dev@openjdk.java.net)
+  * Javac: [`compiler-dev@openjdk.java.net`](mailto:compiler-dev@openjdk.java.net)
+  * Javadoc: [`javadoc-dev@openjdk.java.net`](mailto:javadoc-dev@openjdk.java.net)
+  * JShell: [`kulla-dev@openjdk.java.net`](mailto:kulla-dev@openjdk.java.net)
+  * Panama Project: [`panama-dev@openjdk.java.net`](mailto:panama-dev@openjdk.java.net)
+  * Valhalla Project: [`valhalla-dev@openjdk.java.net`](mailto:valhalla-dev@openjdk.java.net)
+* Security: [`security-dev@openjdk.java.net`](mailto:security-dev@openjdk.java.net)
+
+## Directory to area mapping
+
+* `make` – Build team
+* `hotspot` – HotSpot
+  * `cpu` – Compiler, Runtime
+  * `jdk.*` – Compiler
+  * `os` – Runtime
+  * `os_cpu` – Compiler
+  * `share`
+    * `adlc` – Compiler
+    * `aot` – Compiler
+    * `asm` – Runtime
+    * `c1` – Compiler
+    * `ci` – Compiler
+    * `classfile` – Runtime
+    * `code` – Compiler
+    * `compiler` – Compiler
+    * `gc` – GC
+    * `include` – HotSpot
+    * `interpreter` – Runtime
+    * `jfr` – Runtime
+    * `jvmci` – Compiler
+    * `libadt` – Compiler
+    * `logging` – Runtime
+    * `memory` – Runtime, GC
+    * `metaprogramming` – Runtime
+    * `oops` – Runtime
+    * `opto` – Compiler
+    * `precompiled` – Runtime
+    * `prims` – Runtime, Serviceability
+    * `runtime` – Runtime
+    * `services` – Runtime
+    * `shark` – Compiler
+    * `trace` – Runtime
+    * `utilities` – Runtime
+* `java.base`
+  * Core Libs should almost always be included but Langtools, HotSpot, and/or Security may also be involved.
+  * `classes`
+    * `crypto` – Security
+    * `internal` – HotSpot, Core Libs
+    * `invoke` – Core Libs
+    * `io` – NIO
+    * `lang` – Core Libs
+    * `launcher` – Langtools
+    * `math` – Core Libs
+    * `net` – Net
+    * `nio` – NIO
+    * `reflect` – Core Libs
+    * `security` – Security
+    * `text` – Core Libs
+    * `time` – Core Libs
+    * `util` – I18n, Core Libs
+  * `conf`
+    * `sdp` – Net
+    * `security` – Security
+  * `legal` –
+  * `lib/security` – Security
+  * `man` –
```

Comment:
This depends on who owns the tool.  I believe that this entry corresponds to this part of the repo:

https://hg.openjdk.java.net/jdk/jdk/file/ec0cabbc527f/src/java.base/share/man

I see that there are man pages for java, jfr, and keytool.

Further down in this list I see that mail for the launcher is being directed to LangTools, so that's where I'd send changes for the java man page.

Changes to the jdk.jfr module are being sent to Runtime, so that's where changes to the corresponding man page should be sent.

keytool is owned by Security.

Reply:
Fixed.